skema/rest/workflows.py

```diff
@@ -74,15 +80,21 @@ async def equations_to_amr(data: schema.EquationLatexToAMR):
     r.json()
     """
     mml: List[str] = [
-        utils.clean_mml(eqn2mml.get_mathml_from_latex(tex))
+        utils.clean_mml(
+            eqn2mml.get_mathml_from_latex(eqn2mml.LatexEquation(tex_src=tex))
```

Comment:
This is now out of sync with `main`...

Reply:
It should be 

```python
utils.clean_mml(eqn2mml.get_mathml_from_latex(tex))
```

Reply:
See https://github.com/ml4ai/skema/blob/2a56307416f5628572317de8ef49f6f6de5e5cba/skema/rest/workflows.py#L77

merging after running black makes conflicts tricky...

Reply:
I edited the file to look like it used too. Can you please double check? Better safe than sorry ...